src/octoprint/static/js/app/helpers.js

```diff
@@ -1685,10 +1685,11 @@ var negotiateWebRTC = function (streamUrl) {
         });
 };
 
-var startWebRTC = function (videoElement, streamUrl) {
+var startWebRTC = function (videoElement, streamUrl, iceServers) {
+    iceServers = iceServers || ["stun:stun.l.google.com:19302"];
     var config = {
         sdpSemantics: "unified-plan",
-        iceServers: [{urls: ["stun:stun.l.google.com:19302"]}]
+        iceServers: [{urls: iceServers}]
```

Comment:
If you make this optional, you should probably break this into a separate check and don't set `iceServers` at all if the setting is unset.

For example:
https://github.com/aiortc/aiortc/blob/main/examples/webcam/client.js#L49